bundles/binding/org.openhab.binding.dmx.e131/src/main/java/org/openhab/binding/dmx/e131/internal/E131Node.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) 2010-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.dmx.e131.internal;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+/**
+ * E1.31 node class
+ *
+ * this is a receiving or sending E1.31 node with address and port
+ *
+ * @author Jan N. Klug
+ * @since 1.9.0
+ */
+
+public class E131Node {
+
+    public static final int DEFAULT_PORT = 5568;
+
+    protected int port;
+    protected InetAddress address;
+
+    /**
+     * default constructor
+     */
+    public E131Node() {
+        this.port = DEFAULT_PORT;
+    }
+
+    /**
+     * constructor with address
+     *
+     * @param addrString
+     *            domain name or IP address as string representation
+     */
+    public E131Node(String addrString) {
+        this.port = DEFAULT_PORT;
+        setInetAddress(addrString);
+    }
+
+    /**
+     * constructor with address and port
+     *
+     * @param addrString
+     *            domain name or IP address as string representation
+     * @param port
+     *            UDP port of the receiver node
+     */
+    public E131Node(String addrString, int port) {
+        this.port = port;
+        setInetAddress(addrString);
+    }
+
+    /**
+     * sets the node address
+     *
+     * @param addrString
+     *            domain name or IP address as string representation
+     */
+    public void setInetAddress(String addrString) {
+        try {
+            this.address = InetAddress.getByName(addrString);
+        } catch (UnknownHostException e) {
+
```

Comment:
Would be a good spot for warn log entry, since the old value of `this.address` would remain unchanged on exception.